packages/features/shell/Shell.tsx

```diff
@@ -782,8 +782,8 @@ export function ShellMain(props: LayoutProps) {
     <>
       <div
         className={classNames(
-          "flex items-center md:mt-0 md:mb-6",
-          props.smallHeading ? "lg:mb-7" : "lg:mb-8"
+          "flex items-center md:mt-0 md:mb-8",
+          props.smallHeading ? "lg:mb-9" : "lg:mb-10"
```

Comment:
Looks like routing forms has a larger gap between shell and the list than for example`/event-types` or `/bookings`  ([Figma Design](https://www.figma.com/file/xk4HOxtSI82J0F7enMxeak/Cal---Live?node-id=20%3A15579&t=yqnXxCFaOm5sFRs0-0))

So let's not make the changes in the shell itself and instead add it directly to routing forms. @Jaibles Or should we make the gap below the shell the same as it is for event types? I think workflows has a larger gap too

Reply:
What should we do @Jaibles?

Reply:
There should be a space of 32px.
<img width="1261" alt="CleanShot 2023-03-06 at 17 54 03@2x" src="https://user-images.githubusercontent.com/4536123/223191489-48903c66-2d5d-481e-90eb-12d7b5e093f6.png">

![CleanShot 2023-03-06 at 17 53 51@2x](https://user-images.githubusercontent.com/4536123/223191500-7220a6f7-3ed1-4a4f-a565-6dd18e16420d.png)

Reply:
Thank you! @gitstart means we should kee `mb` as it is. Once this is fixed I can approve 🙏